flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskTest.java

```diff
@@ -540,9 +527,9 @@ public void testFailExternallyDuringInvoke() throws Exception {
 
         task.getExecutingThread().join();
 
-        assertEquals(ExecutionState.FAILED, task.getExecutionState());
-        assertTrue(task.isCanceledOrFailed());
-        assertTrue(task.getFailureCause().getMessage().contains("test"));
+        assertThat(task.getExecutionState()).isEqualTo(ExecutionState.FAILED);
+        assertThat(task.isCanceledOrFailed()).isTrue();
+        assertThat(task.getFailureCause().getMessage().contains("test")).isTrue();
```

Comment:
we use `contains(XX)` interface directly, you can check it globally

Reply:
fixed